docs/releases.rst

```diff
@@ -4,6 +4,16 @@
     Releases
 ======================
 
+tmt-1.49.0
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Output of the :ref:`/plugins/report/display` and
+:ref:`/plugins/report/display` is changing in this release, to provide
```

Comment:
The same plugin is reported twice, seems like a copy & paste typo?

Reply:
Yes, it's a typo. Fixed in 95de5a01.